runtime/ts/bytes.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+/**
+ * Represents a Blob of binary data. Intended for use in storing things such as
+ * media data.
+ */
+
+import {encode, decode} from './keymgmt/base64.js';
+
+export class Bytes {
+  private blob: Uint8Array;
+
+  /**
+   * Construct a blob from a base64 string
+   */
+  constructor(base64bytes: string) {
```

Comment:
FTR I think the original plan was for at least the web variant ctor of this to use an actual `Blob` rather than a Base64 string as the browser has some perf optimizations around doing so. See the `Background` section here: https://docs.google.com/document/d/1245YKcJHaQEZxCQw2ozzFyyPl31_mSup8azqDOeVCSs/edit#heading=h.vna3mbgb7pf4

I'm ok with the change as is but would at least add a TODO noting.

Reply:
Blob type can be constructed from Uint8Array

```
   const b = new Blob([arraybuf]);
```

should we switch to that as the API?